mods/d2k/rules/vehicles.yaml

```diff
@@ -442,7 +442,7 @@ deviator:
 		BuildPaletteOrder: 40
 		BuildDuration: 373
 		BuildDurationModifier: 40
-		Description: Main Battle Tank\n  Strong vs Tanks\n  Weak vs Infantry, Aircraft\n \n  Atreides:      +Range\n  Harkonnen:  +Health\n  Ordos:          +Speed
+		Description: Main Battle Tank\n  Strong vs Tanks\n  Weak vs Infantry
```

Comment:
No problem :) PRs are not running away and I think it is good to add them under their description.

Still I wanted to display the differences between the tanks but this is not a balancing PR, still maybe someone can use it someday.

Faction | Armor/Health | Firepower | Speed*
|:----------:|:---------------:|:---------------:|:---------------:|
Atreides| + | + | + 
Harkonnen| ++ | ++ | --
Ordos| -- | -- | ++


*Speed: Vehiclespeed, Turnspeed, Turret-Turnspeed